test/e2e/features/proxy.feature

```diff
@@ -9,8 +9,8 @@ Feature: Behind proxy test
 
     Scenario: Start CRC
         Given executing crc setup command succeeds
-        And  executing "crc config set http-proxy http://192.168.130.1:3128" succeeds
-        Then executing "crc config set https-proxy http://192.168.130.1:3128" succeeds
+        And  executing crc config set http-proxy http://192.168.130.1:3128 command succeeds
```

Comment:
I am not a big fan of removing the quote marks as it makes reading the test much harder... as you are leaving commands like `oc` intact.

Reply:
There currently is a mix of `And execute crc setup` and `And executing "crc setup"`, so this at least adds consistency.
I'm not sure it's possible to automatically add "--log-level debug" while keeping the quote marks. And readability/consistency is going to be quite bad if we decide to add it explicitly to each `And executing "crc --log-level debug setup"`